tools/windows-node-installer/pkg/cloudprovider/interface.go

```diff
@@ -3,13 +3,11 @@ package cloudprovider
 // Cloud is the interface that needs to be implemented per provider to allow support for creating Windows nodes on
 // that provider.
 type Cloud interface {
-	// CreateWindowsVM creates windows VM based on available image id, instance type, and key name
-	// TODO: CreateWindowsVM should return a vendor session for further interaction with the created instance
-	CreateWindowsVM(imageId, instanceType, keyName string) error
-	// DestroyWindowsVM uses 'windows-node-installer.json' file which contains information such as ID's and
-	// security groups of created instances and deletes them.
-	// It deletes the security groups only if the group is not associated with any instance.
-	// This function can also serve as a clean up operation for destroying instances that failed to be completely
-	// created.
+	// CreateWindowsVM creates a Windows VM based on available image id, instance type, and ssh key name.
+	// TODO: CreateWindowsVM should return a vendor session for further interaction with the created instance.
+	CreateWindowsVM(imageId, instanceType, sshKey string) error
+	// DestroyWindowsVM uses 'windows-node-installer.json' file that contains IDs of created instance and
+	// security group and deletes them.
```

Comment:
Like @ravisantoshgudimetla and I mentioned before, you need to clearly document the structure of this JSON file.